Documentation/Global/Constant.rst

```diff
@@ -12,5 +8,72 @@
 Constant ViewHelper `<f:constant>`
 ==================================
 
+..  note::
+    The constant ViewHelper can display **PHP constants** only. It cannot be
+    used to display `TypoScript Constants <https://docs.typo3.org/permalink/t3tsref:typoscript-syntax-constants>`_
+    or `Site settings <https://docs.typo3.org/permalink/t3coreapi:sitehandling-settings>`_.
+
+..  typo3:viewhelper:: constant
+    :source: ../Global.json
```

Comment:
In the other patch this file was removed

Reply:
No, will not be removed or changed, I just auto-formatted it and accidently committed in the other PR. The json is automatically updated